### PR TITLE
Remove abandoned box component

### PR DESCRIPTION
**Pull request by timomeh**
### 1. Remove Box Component

The `<Box />` component was used a lot of times, but almost every time it was used as a plain `<div />`. The main feature of `<Box />` were flex-props which would be translated to inline styles. This was used very rarely. So I removed the Box Component and all its instances.

### 2. Use mounting element in index.ejs

The React App(s) were rendered by creating a div, appending it to the body, adding a css-modules class to it, and mounting the React App into this div.

It's common practice (see CRA, Next.js, Gatsby) to have an empty element with an ID inside the static html file and mount the React App into it. ...and I did that!

### 3. Remove LanguageSwitcher

Seems like `LanguageSwitcher` was abandoned, so I removed it. It was the only file which had the dependency of `scratch-l10n`, thus I uninstalled that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,7 +126,6 @@
     "request": "^2.88.0",
     "request-promise-native": "^1.0.5",
     "rimraf": "^2.6.1",
-    "scratch-l10n": "3.1.20190220143732",
```
Comment (danieldiekmeier) [inline package.json:129]: Hell yes, this is great

File: src/components/gui/gui.jsx [hunk 2]
```diff
@@ -131,19 +130,21 @@ const GUIComponent = (props) => {
...
-          <Box
-            direction="row"
-            justifyContent="center"
-            style={{ display: 'flex' }}
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              justifyContent: 'center',
+            }}
```
Comment (danieldiekmeier) [inline src/components/gui/gui.jsx:147]: 🥰

File: src/components/loupe/loupe.jsx [hunk 3]
```diff
@@ -66,21 +65,19 @@ class LoupeComponent extends React.Component {
     this.canvas = element
   }
   render() {
-    const { colorInfo, ...boxProps } = this.props
+    const { colorInfo } = this.props
     return (
-      <Box
-        {...boxProps}
+      <canvas
```
Comment (danieldiekmeier) [inline src/components/loupe/loupe.jsx:70]: lel

File: src/containers/app.jsx [hunk 4]
```diff
@@ -185,13 +185,11 @@ class App extends Component {
...
-          <IntlProvider locale="de" messages={localeDe}>
-            <div className={styles.app}>{this.renderView()}</div>
-          </IntlProvider>
-        </MobileScreenFallback>
-      </>
+      <MobileScreenFallback>
+        <IntlProvider locale="de" messages={localeDe}>
+          <div className={styles.app}>{this.renderView()}</div>
+        </IntlProvider>
+      </MobileScreenFallback>
```
Comment (danieldiekmeier) [inline src/containers/app.jsx:192]: Oh, my bad. Thanks for catching this! In the earlier version, the `MobileScreen` was next to the `IntlProvider` and the fragment was required.

File: src/entrypoints/index.ejs [hunk 5]
```diff
@@ -64,6 +64,7 @@
     <title><%= options.title %></title>
   </head>
   <body>
+    <div id="__mausapp"></div>
```
Comment (danieldiekmeier) [inline src/entrypoints/index.ejs:67]: Awesome! I thought about doing this while I changed the `MobileScreenFallback`, but I thought that the PR was already big enough. Thanks!

**Review by danieldiekmeier — APPROVED** [review]

File: src/containers/app.jsx (hunk 4 above)
Comment (timomeh) [reply src/containers/app.jsx:192]: I found an eslint rule for that: [react/jsx-no-useless-fragment](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md) 🎉 Should I enable it?

Comment (danieldiekmeier) [reply src/containers/app.jsx:192]: Yes, sounds like a good rule!

**Pull request merged**